src/main/default/lwc/voteCount/__test__/voteCount.test.js

```diff
@@ -0,0 +1,122 @@
+import { createElement } from 'lwc';
+import VoteCount from 'c/voteCount';
+
+function createTestComponent() {
+  const element = createElement('c-vote-count', {
+    is: VoteCount
+  });
+  document.body.appendChild(element);
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers_plural',
+  () => ({ default: 'Helped 102 users' }),
+  {
+    virtual: true
+  }
+);
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers',
+  () => ({ default: 'Helped 1 user' }),
+  {
+    virtual: true
+  }
+);
+
+jest.mock(
+  '@salesforce/label/c.cookbook_HelpedUsers_zero',
+  () => ({ default: 'Helped no one' }),
```

Comment:
"Helped no one" sounds passive agressive 😂

Reply:
Maybe something like "This document has not been rated yet" would be more friendly. I feel "rated" might not be the right word though.

@acarpentier01 Do you have an idea to improve this message?

Reply:
How about: "Be the first to rate this document." or "No user ratings available."